### PR TITLE
Removed unused methods and imports.

### PR DESCRIPTION
**Pull request by mchf**

### CODE REVIEW

File: src/include/network/services/dns.rb [hunk 1]
```diff
@@ -645,10 +645,6 @@ def storeHostnameGlobal(_key, event)
       nil
     end
 
-    def ReallyAbortInst
-      Popup.ConfirmAbort(:incomplete)
```
Comment (teclator) [inline src/include/network/services/dns.rb:649]: It is used here https://github.com/yast/yast-firstboot/blob/master/src/lib/y2firstboot/clients/hostname.rb#L97

Comment (mchf) [reply src/include/network/services/dns.rb:649]: so it is time to move it there, isn't it?

**Comment by mchf** [comment]
respective change is in TW already

**Review by teclator — APPROVED** [review]
With [yast2-firstboot-4.4.2](https://build.opensuse.org/request/show/903243) It looks safer to merge this PR.

**Pull request merged**